packages/dataviews/package.json

```diff
@@ -38,10 +38,14 @@
 	"react-native": "src/index",
 	"wpScript": true,
 	"types": "build-types",
-	"sideEffects": false,
+	"sideEffects": [
```

Comment:
What is this for?

Reply:
We're adding sideEffects to the package.json to ensure CSS/SCSS files are properly loaded in Storybook while allowing webpack to optimize other modules through tree-shaking

Reply:
Upstream does this differently, by loading the [built CSS files](https://github.com/WordPress/gutenberg/pull/56761/files). Can we change the storybook configuration to do the same and not changing `sideEffects: false` in this `package.json` file?

Reply:
gotcha. Yes, let’s try to stick to the core den-env setup as much as possible